js/index.js

```diff
@@ -1,7 +1,46 @@
-// Iteration 1: Names and Input
+let hacker1 = "Gosia";
+let hacker2 = "Hugo";
 
+for (let i=0; i < hacker2.length; i++) {
+  console.log(hacker2[i].toUpperCase());
+}
```

Comment:
In here, you could use the template literal syntax (the backticks) in order to print both names to the console!
Ex: console.log(`"The driver's name is ${insert variable with the drivers name here}"`